addons/addons.xml

```diff
@@ -15,7 +15,7 @@
     <name>AEcho</name>
     <description>Automatically uses echo drops when you get silenced. Also, uses send to send a message to an alt that you got debuffed.</description>
     <author>Nitrous (Shiva)</author>
-    <support>https://discord.gg/b275nMv</support>
+    <support>https://discord.gg/aUrHCvk</support>
```

Comment:
I really appreciate the intent since we talked about this on Discord, but please try to limit pull requests to specific concerns. This has nothing to do with the addon and does not belong in here. I mentioned this last time as well when you made a PR for multiple addons.

Aside from better organization and cleaner commit history, this has tangible benefits, if something broke in this commit we cannot easily revert it without removing your entire addon. That puts us on the spot and requires us to fix, test and deploy it quickly. I'll accept it this time because the changes are pretty minor, but please pay attention to that in the future.

Reply:
Excellent point. Totally agree.